vault/activity_log.go

```diff
@@ -1889,6 +1895,23 @@ func newProcessCounts() *processCounts {
 	}
 }
 
+func (p *processCounts) add(client *activity.EntityRecord) {
+	if client.NonEntity {
+		p.NonEntities[client.ClientID] = struct{}{}
+	} else {
+		p.Entities[client.ClientID] = struct{}{}
+	}
+}
+
+func (p *processCounts) contains(client *activity.EntityRecord) bool {
```

Comment:
Where is this used? I couldn't find a reference to it.

Reply:
It's actually not used here - it's used in testing in a later change that's on top of this one. However, since it relates to the code here, I included it. Let me know if you would prefer that I remove it and add it to a later PR

Reply:
Ah, cool! Thanks for the context!